Add-ons/Umbraco-Plumber/Workflow-Content-App/index.md

```diff
@@ -0,0 +1,87 @@
+
+When the current node is pending workflow approval, the **Active workflow** sub-section displays detailed information such as:
+
+- Approve, reject, or cancel pending workflow tasks.
+- View change description and track differences across pending and completed workflows.
+- View the group responsible for approving the pending workflow.
+- View pending language variant(s) workflow.
+- View the workflow activity (eg pending approval/task approvals/rejects) for the current workflow process.
+
+![Active Workflow sub-section](images/Active_Workflow_detailed_info.png)
+
+You can access Active Workflows from two places - the **Content** section and the **Workflow** section (depending on your user permission). Workflow Administrators (those users with access to the Workflow section) can action workflows assigned to a different group. In the **Workflow History**, these are noted as being performed by the admin.
+
+In multi-lingual sites, variant content can be submitted in one of these workflows:
+
+- Only the current variant.
+- All variants for publishing in a single workflow process using the workflow applied to the default variant.
+- Each variant into a separate workflow.
+
+For example, you can have the German version of your content approved by groups of German speakers and the English version by the English speaker's group.
+
+## Configuration
+
+The Configuration sub-section provides an interface for configuring the content approval flow for the current node and displays any inherited or document-type approval flows applied to the current content node.
+
+![Configuration sub-section](images/Configuration-sub-section.png)
+
+### Content Approval Flow
+
+You can add different groups for different stages of content approval flow. Content Approval flow groups can be reordered via drag and drop. You can also apply the approval flow either for publish and unpublish workflow or only publish workflow.
+
+![Content approval flow](images/content-approval-flow.gif)
+
+### Approval Flow Types
+
+Approval Flows are available in three types: Content approval flow, Inherited approval flow, and Document-type approval flow.
+
+A given content node may have all three approval flow types applied but only one will be applied as per the following order of priority:
+
+- **Content approval flow:** set directly on a content node via the **Configuration** section in the **Workflow** tab . This type will take priority over all others.
+- **Inherited approval flow:** if a node has no Content approval flow set, nor a flow applied to its Document Type, Umbraco Plumber will traverse the content tree until it finds a node with a Content approval flow and will use this flow for the current change.
+- **Document-type approval flow:** set in the **Settings** section. This approval flow will apply to all content nodes of the selected Document Type unless the node has a Content approval flow set. This feature requires a license.
```

Comment:
Inherited should be last, as it's the fallback after checking content and doc-type flow